EventFilter/HcalRawToDigi/plugins/HcalRawToDigi.cc

```diff
@@ -65,6 +65,7 @@ HcalRawToDigi::HcalRawToDigi(edm::ParameterSet const& conf):
     produces<HcalUMNioDigi>();
   produces<QIE10DigiCollection>();
   produces<QIE11DigiCollection>();
+  produces<QIE10DigiCollection>("ZDC");
```

Comment:
there is no matching "keep" in RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py
I suppose it should be added everywhere we have ```keep ZDC*```
without this the created products will be lost